2024-05-14 00:40:58.090 [background-preinit] INFO  o.h.validator.internal.util.Version - HV000001: Hibernate Validator 8.0.1.Final
2024-05-14 00:40:58.128 [main] INFO  c.e.p.e.EscuelasProjectApplicationTests - Starting EscuelasProjectApplicationTests using Java 17.0.9 with PID 10684 (started by Usuario in G:\Desktop\EscuelasProject\escuelas_project)
2024-05-14 00:40:58.129 [main] INFO  c.e.p.e.EscuelasProjectApplicationTests - No active profile set, falling back to 1 default profile: "default"
2024-05-14 00:40:59.364 [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2024-05-14 00:40:59.456 [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 82 ms. Found 5 JPA repository interfaces.
2024-05-14 00:41:00.405 [main] INFO  o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [name: default]
2024-05-14 00:41:00.507 [main] INFO  org.hibernate.Version - HHH000412: Hibernate ORM core version 6.4.4.Final
2024-05-14 00:41:00.563 [main] INFO  o.h.c.i.RegionFactoryInitiator - HHH000026: Second-level cache disabled
2024-05-14 00:41:00.932 [main] INFO  o.s.o.j.p.SpringPersistenceUnitInfo - No LoadTimeWeaver setup: ignoring JPA class transformer
2024-05-14 00:41:00.972 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
2024-05-14 00:41:01.320 [main] INFO  com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Added connection com.mysql.cj.jdbc.ConnectionImpl@77204651
2024-05-14 00:41:01.323 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2024-05-14 00:41:02.478 [main] INFO  o.h.e.t.j.p.i.JtaPlatformInitiator - HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2024-05-14 00:41:02.567 [main] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
2024-05-14 00:41:03.240 [main] INFO  o.s.d.j.r.query.QueryEnhancerFactory - Hibernate is in classpath; If applicable, HQL parser will be used.
2024-05-14 00:41:05.636 [main] INFO  o.s.b.t.m.w.SpringBootMockServletContext - Initializing Spring TestDispatcherServlet ''
2024-05-14 00:41:05.636 [main] INFO  o.s.t.w.s.TestDispatcherServlet - Initializing Servlet ''
2024-05-14 00:41:05.640 [main] INFO  o.s.t.w.s.TestDispatcherServlet - Completed initialization in 1 ms
2024-05-14 00:41:05.710 [main] INFO  c.e.p.e.EscuelasProjectApplicationTests - Started EscuelasProjectApplicationTests in 8.08 seconds (process running for 9.454)
2024-05-14 00:41:06.415 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), URL: http://localhost/api/v1/escuelas/enable/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:06.509 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), Respuesta: <200 OK OK,ResponseMessage[message=ha habilitado exitosamente el escuela con el id: 1, errorCode=0],[]>
2024-05-14 00:41:06.666 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), URL: http://localhost/api/v1/escuelas/enable/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:06.670 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela ya estaba habilitada]
2024-05-14 00:41:06.670 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela ya estaba habilitada, errorCode=2],[]>
2024-05-14 00:41:06.682 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 00:41:06.682 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 00:41:06.695 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), URL: http://localhost/api/v1/escuelas/disable/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:06.697 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), Respuesta: <200 OK OK,ResponseMessage[message=Se ha eliminado exitosamente el escuela con el id: 3, errorCode=0],[]>
2024-05-14 00:41:06.709 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), URL: http://localhost/api/v1/escuelas/find/99999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:06.718 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 00:41:06.718 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 00:41:06.728 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 00:41:06.728 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 00:41:06.738 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), URL: http://localhost/api/v1/escuelas/find/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:06.743 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), Respuesta: <200 OK OK,EscuelaResponseDto[id=3, nombre=test],[]>
2024-05-14 00:41:06.756 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 00:41:06.756 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 00:41:06.765 [main] WARN  o.s.web.servlet.PageNotFound - No mapping for DELETE /api/v1/escuelas/disable/
2024-05-14 00:41:06.768 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationErrorRutaNoMapeada, Arguments: [org.springframework.web.servlet.NoHandlerFoundException: No endpoint DELETE /api/v1/escuelas/disable/.]
2024-05-14 00:41:06.768 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationErrorRutaNoMapeada(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La ruta o endpoint que desea acceder no existe, errorCode=6],[]>
2024-05-14 00:41:06.866 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.save(..), URL: http://localhost/api/v1/escuelas/save, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:06.967 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.save(..), Respuesta: <201 CREATED Created,EscuelaResponseDto[id=15, nombre=test2],[]>
2024-05-14 00:41:06.984 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 00:41:06.984 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 00:41:06.994 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public org.springframework.http.ResponseEntity<com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaResponseDto> com.escuelas.project.escuelas_project.controllers.v1.EscuelaController.update(com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaDtoUpdate,java.lang.Long) throws com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException,com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException]
2024-05-14 00:41:06.994 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El body debe estar en formato JSON, errorCode=1],[]>
2024-05-14 00:41:07.005 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), URL: http://localhost/api/v1/escuelas/update/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:07.008 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela que busca se encuentra deshabilitada]
2024-05-14 00:41:07.008 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela que busca se encuentra deshabilitada, errorCode=2],[]>
2024-05-14 00:41:07.017 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), URL: http://localhost/api/v1/escuelas/disable/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:07.030 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela ya estaba deshabilitada]
2024-05-14 00:41:07.030 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela ya estaba deshabilitada, errorCode=2],[]>
2024-05-14 00:41:07.039 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), URL: http://localhost/api/v1/escuelas/disable/9999999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:07.043 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 00:41:07.044 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 00:41:07.056 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), URL: http://localhost/api/v1/escuelas/update/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:07.058 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), Respuesta: <200 OK OK,EscuelaResponseDto[id=3, nombre=test2 temporal],[]>
2024-05-14 00:41:07.068 [main] WARN  o.s.web.servlet.PageNotFound - No mapping for PUT /api/v1/escuelas/enable/
2024-05-14 00:41:07.069 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationErrorRutaNoMapeada, Arguments: [org.springframework.web.servlet.NoHandlerFoundException: No endpoint PUT /api/v1/escuelas/enable/.]
2024-05-14 00:41:07.069 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationErrorRutaNoMapeada(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La ruta o endpoint que desea acceder no existe, errorCode=6],[]>
2024-05-14 00:41:07.078 [main] WARN  o.s.web.servlet.PageNotFound - No mapping for PUT /api/v1/escuelas/update/
2024-05-14 00:41:07.079 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationErrorRutaNoMapeada, Arguments: [org.springframework.web.servlet.NoHandlerFoundException: No endpoint PUT /api/v1/escuelas/update/.]
2024-05-14 00:41:07.079 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationErrorRutaNoMapeada(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La ruta o endpoint que desea acceder no existe, errorCode=6],[]>
2024-05-14 00:41:07.088 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), URL: http://localhost/api/v1/escuelas/find/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:07.094 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela que busca se encuentra deshabilitada]
2024-05-14 00:41:07.095 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela que busca se encuentra deshabilitada, errorCode=2],[]>
2024-05-14 00:41:07.103 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), URL: http://localhost/api/v1/escuelas/enable/9999999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:07.105 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 00:41:07.105 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 00:41:07.117 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), URL: http://localhost/api/v1/escuelas/update/9999999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:07.119 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 00:41:07.120 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 00:41:07.168 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaResponseDto> com.escuelas.project.escuelas_project.controllers.v1.EscuelaController.save(com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaDto) throws com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaExistenteException: [Field error in object 'escuelaDto' on field 'nombre': rejected value []; codes [NotBlank.escuelaDto.nombre,NotBlank.nombre,NotBlank.java.lang.String,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [escuelaDto.nombre,nombre]; arguments []; default message [nombre]]; default message [El nombre de la escuela es requerido para la creacion de una nueva escuela]] ]
2024-05-14 00:41:07.169 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,[ResponseMessage[message=El nombre de la escuela es requerido para la creacion de una nueva escuela, errorCode=1]],[]>
2024-05-14 00:41:07.185 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.save(..), URL: http://localhost/api/v1/escuelas/save, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:07.190 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaExistenteException: La escuela ya existe en la base de datos]
2024-05-14 00:41:07.190 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaExistenteException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=No se pudo registrar el escuela con el nombre ingresado ya que la misma ya se encuentra registrada, errorCode=4],[]>
2024-05-14 00:41:07.198 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findAll(), URL: http://localhost/api/v1/escuelas/find/all, IP: 127.0.0.1, Parámetros: {}
2024-05-14 00:41:07.206 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findAll(), Respuesta: <200 OK OK,[EscuelaResponseDto[id=1, nombre=test_d], EscuelaResponseDto[id=2, nombre=TEC 1], EscuelaResponseDto[id=3, nombre=test]],[]>
2024-05-14 00:41:07.221 [SpringApplicationShutdownHook] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
2024-05-14 00:41:07.224 [SpringApplicationShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
2024-05-14 00:41:07.235 [SpringApplicationShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
2024-05-14 09:10:33.653 [background-preinit] INFO  o.h.validator.internal.util.Version - HV000001: Hibernate Validator 8.0.1.Final
2024-05-14 09:10:33.693 [main] INFO  c.e.p.e.EscuelasProjectApplicationTests - Starting EscuelasProjectApplicationTests using Java 17.0.9 with PID 9472 (started by Usuario in G:\Desktop\EscuelasProject\escuelas_project)
2024-05-14 09:10:33.694 [main] INFO  c.e.p.e.EscuelasProjectApplicationTests - No active profile set, falling back to 1 default profile: "default"
2024-05-14 09:10:34.774 [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2024-05-14 09:10:34.882 [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 98 ms. Found 5 JPA repository interfaces.
2024-05-14 09:10:35.775 [main] INFO  o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [name: default]
2024-05-14 09:10:35.869 [main] INFO  org.hibernate.Version - HHH000412: Hibernate ORM core version 6.4.4.Final
2024-05-14 09:10:35.931 [main] INFO  o.h.c.i.RegionFactoryInitiator - HHH000026: Second-level cache disabled
2024-05-14 09:10:36.378 [main] INFO  o.s.o.j.p.SpringPersistenceUnitInfo - No LoadTimeWeaver setup: ignoring JPA class transformer
2024-05-14 09:10:36.418 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
2024-05-14 09:10:36.757 [main] INFO  com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Added connection com.mysql.cj.jdbc.ConnectionImpl@c190cfc
2024-05-14 09:10:36.760 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2024-05-14 09:10:38.068 [main] INFO  o.h.e.t.j.p.i.JtaPlatformInitiator - HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2024-05-14 09:10:38.162 [main] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
2024-05-14 09:10:38.717 [main] INFO  o.s.d.j.r.query.QueryEnhancerFactory - Hibernate is in classpath; If applicable, HQL parser will be used.
2024-05-14 09:10:41.152 [main] INFO  o.s.b.t.m.w.SpringBootMockServletContext - Initializing Spring TestDispatcherServlet ''
2024-05-14 09:10:41.153 [main] INFO  o.s.t.w.s.TestDispatcherServlet - Initializing Servlet ''
2024-05-14 09:10:41.157 [main] INFO  o.s.t.w.s.TestDispatcherServlet - Completed initialization in 2 ms
2024-05-14 09:10:41.249 [main] INFO  c.e.p.e.EscuelasProjectApplicationTests - Started EscuelasProjectApplicationTests in 7.995 seconds (process running for 9.135)
2024-05-14 09:10:41.918 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), URL: http://localhost/api/v1/escuelas/enable/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.014 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), Respuesta: <200 OK OK,ResponseMessage[message=ha habilitado exitosamente el escuela con el id: 1, errorCode=0],[]>
2024-05-14 09:10:42.157 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), URL: http://localhost/api/v1/escuelas/enable/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.163 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela ya estaba habilitada]
2024-05-14 09:10:42.163 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela ya estaba habilitada, errorCode=2],[]>
2024-05-14 09:10:42.176 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 09:10:42.176 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 09:10:42.188 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), URL: http://localhost/api/v1/escuelas/disable/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.190 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), Respuesta: <200 OK OK,ResponseMessage[message=Se ha eliminado exitosamente el escuela con el id: 3, errorCode=0],[]>
2024-05-14 09:10:42.200 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), URL: http://localhost/api/v1/escuelas/find/99999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.208 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 09:10:42.208 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 09:10:42.217 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 09:10:42.217 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 09:10:42.227 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), URL: http://localhost/api/v1/escuelas/find/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.233 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), Respuesta: <200 OK OK,EscuelaResponseDto[id=3, nombre=test],[]>
2024-05-14 09:10:42.244 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 09:10:42.244 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 09:10:42.254 [main] WARN  o.s.web.servlet.PageNotFound - No mapping for DELETE /api/v1/escuelas/disable/
2024-05-14 09:10:42.256 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationErrorRutaNoMapeada, Arguments: [org.springframework.web.servlet.NoHandlerFoundException: No endpoint DELETE /api/v1/escuelas/disable/.]
2024-05-14 09:10:42.256 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationErrorRutaNoMapeada(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La ruta o endpoint que desea acceder no existe, errorCode=6],[]>
2024-05-14 09:10:42.359 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.save(..), URL: http://localhost/api/v1/escuelas/save, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.462 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.save(..), Respuesta: <201 CREATED Created,EscuelaResponseDto[id=16, nombre=test2],[]>
2024-05-14 09:10:42.490 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 09:10:42.490 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 09:10:42.499 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public org.springframework.http.ResponseEntity<com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaResponseDto> com.escuelas.project.escuelas_project.controllers.v1.EscuelaController.update(com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaDtoUpdate,java.lang.Long) throws com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException,com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException]
2024-05-14 09:10:42.500 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El body debe estar en formato JSON, errorCode=1],[]>
2024-05-14 09:10:42.509 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), URL: http://localhost/api/v1/escuelas/update/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.512 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela que busca se encuentra deshabilitada]
2024-05-14 09:10:42.512 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela que busca se encuentra deshabilitada, errorCode=2],[]>
2024-05-14 09:10:42.523 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), URL: http://localhost/api/v1/escuelas/disable/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.535 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela ya estaba deshabilitada]
2024-05-14 09:10:42.535 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela ya estaba deshabilitada, errorCode=2],[]>
2024-05-14 09:10:42.544 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), URL: http://localhost/api/v1/escuelas/disable/9999999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.548 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 09:10:42.548 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 09:10:42.556 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), URL: http://localhost/api/v1/escuelas/update/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.558 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), Respuesta: <200 OK OK,EscuelaResponseDto[id=3, nombre=test2 temporal],[]>
2024-05-14 09:10:42.567 [main] WARN  o.s.web.servlet.PageNotFound - No mapping for PUT /api/v1/escuelas/enable/
2024-05-14 09:10:42.567 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationErrorRutaNoMapeada, Arguments: [org.springframework.web.servlet.NoHandlerFoundException: No endpoint PUT /api/v1/escuelas/enable/.]
2024-05-14 09:10:42.568 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationErrorRutaNoMapeada(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La ruta o endpoint que desea acceder no existe, errorCode=6],[]>
2024-05-14 09:10:42.576 [main] WARN  o.s.web.servlet.PageNotFound - No mapping for PUT /api/v1/escuelas/update/
2024-05-14 09:10:42.577 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationErrorRutaNoMapeada, Arguments: [org.springframework.web.servlet.NoHandlerFoundException: No endpoint PUT /api/v1/escuelas/update/.]
2024-05-14 09:10:42.577 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationErrorRutaNoMapeada(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La ruta o endpoint que desea acceder no existe, errorCode=6],[]>
2024-05-14 09:10:42.586 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), URL: http://localhost/api/v1/escuelas/find/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.591 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela que busca se encuentra deshabilitada]
2024-05-14 09:10:42.591 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela que busca se encuentra deshabilitada, errorCode=2],[]>
2024-05-14 09:10:42.601 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), URL: http://localhost/api/v1/escuelas/enable/9999999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.604 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 09:10:42.604 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 09:10:42.615 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), URL: http://localhost/api/v1/escuelas/update/9999999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.618 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 09:10:42.618 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 09:10:42.660 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaResponseDto> com.escuelas.project.escuelas_project.controllers.v1.EscuelaController.save(com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaDto) throws com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaExistenteException: [Field error in object 'escuelaDto' on field 'nombre': rejected value []; codes [NotBlank.escuelaDto.nombre,NotBlank.nombre,NotBlank.java.lang.String,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [escuelaDto.nombre,nombre]; arguments []; default message [nombre]]; default message [El nombre de la escuela es requerido para la creacion de una nueva escuela]] ]
2024-05-14 09:10:42.661 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,[ResponseMessage[message=El nombre de la escuela es requerido para la creacion de una nueva escuela, errorCode=1]],[]>
2024-05-14 09:10:42.679 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.save(..), URL: http://localhost/api/v1/escuelas/save, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.686 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaExistenteException: La escuela ya existe en la base de datos]
2024-05-14 09:10:42.686 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaExistenteException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=No se pudo registrar el escuela con el nombre ingresado ya que la misma ya se encuentra registrada, errorCode=4],[]>
2024-05-14 09:10:42.694 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findAll(), URL: http://localhost/api/v1/escuelas/find/all, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:10:42.704 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findAll(), Respuesta: <200 OK OK,[EscuelaResponseDto[id=1, nombre=test_d], EscuelaResponseDto[id=2, nombre=TEC 1], EscuelaResponseDto[id=3, nombre=test]],[]>
2024-05-14 09:10:42.755 [SpringApplicationShutdownHook] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
2024-05-14 09:10:42.761 [SpringApplicationShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
2024-05-14 09:10:42.771 [SpringApplicationShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
2024-05-14 09:11:10.155 [background-preinit] INFO  o.h.validator.internal.util.Version - HV000001: Hibernate Validator 8.0.1.Final
2024-05-14 09:11:10.192 [main] INFO  c.e.p.e.EscuelasProjectApplicationTests - Starting EscuelasProjectApplicationTests using Java 17.0.9 with PID 9840 (started by Usuario in G:\Desktop\EscuelasProject\escuelas_project)
2024-05-14 09:11:10.193 [main] INFO  c.e.p.e.EscuelasProjectApplicationTests - No active profile set, falling back to 1 default profile: "default"
2024-05-14 09:11:11.384 [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2024-05-14 09:11:11.490 [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 95 ms. Found 5 JPA repository interfaces.
2024-05-14 09:11:12.390 [main] INFO  o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [name: default]
2024-05-14 09:11:12.482 [main] INFO  org.hibernate.Version - HHH000412: Hibernate ORM core version 6.4.4.Final
2024-05-14 09:11:12.532 [main] INFO  o.h.c.i.RegionFactoryInitiator - HHH000026: Second-level cache disabled
2024-05-14 09:11:12.943 [main] INFO  o.s.o.j.p.SpringPersistenceUnitInfo - No LoadTimeWeaver setup: ignoring JPA class transformer
2024-05-14 09:11:12.988 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
2024-05-14 09:11:13.346 [main] INFO  com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Added connection com.mysql.cj.jdbc.ConnectionImpl@416c8bb5
2024-05-14 09:11:13.349 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2024-05-14 09:11:14.623 [main] INFO  o.h.e.t.j.p.i.JtaPlatformInitiator - HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2024-05-14 09:11:14.722 [main] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
2024-05-14 09:11:15.456 [main] INFO  o.s.d.j.r.query.QueryEnhancerFactory - Hibernate is in classpath; If applicable, HQL parser will be used.
2024-05-14 09:11:18.210 [main] INFO  o.s.b.t.m.w.SpringBootMockServletContext - Initializing Spring TestDispatcherServlet ''
2024-05-14 09:11:18.212 [main] INFO  o.s.t.w.s.TestDispatcherServlet - Initializing Servlet ''
2024-05-14 09:11:18.217 [main] INFO  o.s.t.w.s.TestDispatcherServlet - Completed initialization in 3 ms
2024-05-14 09:11:18.299 [main] INFO  c.e.p.e.EscuelasProjectApplicationTests - Started EscuelasProjectApplicationTests in 8.55 seconds (process running for 9.705)
2024-05-14 09:11:19.143 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), URL: http://localhost/api/v1/escuelas/enable/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.222 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), Respuesta: <200 OK OK,ResponseMessage[message=ha habilitado exitosamente el escuela con el id: 1, errorCode=0],[]>
2024-05-14 09:11:19.371 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), URL: http://localhost/api/v1/escuelas/enable/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.375 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela ya estaba habilitada]
2024-05-14 09:11:19.376 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela ya estaba habilitada, errorCode=2],[]>
2024-05-14 09:11:19.387 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 09:11:19.388 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 09:11:19.398 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), URL: http://localhost/api/v1/escuelas/disable/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.401 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), Respuesta: <200 OK OK,ResponseMessage[message=Se ha eliminado exitosamente el escuela con el id: 3, errorCode=0],[]>
2024-05-14 09:11:19.412 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), URL: http://localhost/api/v1/escuelas/find/99999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.420 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 09:11:19.420 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 09:11:19.429 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 09:11:19.430 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 09:11:19.439 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), URL: http://localhost/api/v1/escuelas/find/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.444 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), Respuesta: <200 OK OK,EscuelaResponseDto[id=3, nombre=test],[]>
2024-05-14 09:11:19.454 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 09:11:19.455 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 09:11:19.464 [main] WARN  o.s.web.servlet.PageNotFound - No mapping for DELETE /api/v1/escuelas/disable/
2024-05-14 09:11:19.466 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationErrorRutaNoMapeada, Arguments: [org.springframework.web.servlet.NoHandlerFoundException: No endpoint DELETE /api/v1/escuelas/disable/.]
2024-05-14 09:11:19.467 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationErrorRutaNoMapeada(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La ruta o endpoint que desea acceder no existe, errorCode=6],[]>
2024-05-14 09:11:19.551 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.save(..), URL: http://localhost/api/v1/escuelas/save, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.637 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.save(..), Respuesta: <201 CREATED Created,EscuelaResponseDto[id=17, nombre=test2],[]>
2024-05-14 09:11:19.668 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; For input string: "error"]
2024-05-14 09:11:19.668 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El dato enviado por url es invalido, por favor revise la documentacion y los tipos de datos permitdos, errorCode=5],[]>
2024-05-14 09:11:19.680 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public org.springframework.http.ResponseEntity<com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaResponseDto> com.escuelas.project.escuelas_project.controllers.v1.EscuelaController.update(com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaDtoUpdate,java.lang.Long) throws com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException,com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException]
2024-05-14 09:11:19.680 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=El body debe estar en formato JSON, errorCode=1],[]>
2024-05-14 09:11:19.689 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), URL: http://localhost/api/v1/escuelas/update/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.692 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela que busca se encuentra deshabilitada]
2024-05-14 09:11:19.692 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela que busca se encuentra deshabilitada, errorCode=2],[]>
2024-05-14 09:11:19.701 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), URL: http://localhost/api/v1/escuelas/disable/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.714 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela ya estaba deshabilitada]
2024-05-14 09:11:19.715 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela ya estaba deshabilitada, errorCode=2],[]>
2024-05-14 09:11:19.722 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.disable(..), URL: http://localhost/api/v1/escuelas/disable/9999999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.727 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 09:11:19.728 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 09:11:19.739 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), URL: http://localhost/api/v1/escuelas/update/3, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.741 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), Respuesta: <200 OK OK,EscuelaResponseDto[id=3, nombre=test2 temporal],[]>
2024-05-14 09:11:19.752 [main] WARN  o.s.web.servlet.PageNotFound - No mapping for PUT /api/v1/escuelas/enable/
2024-05-14 09:11:19.752 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationErrorRutaNoMapeada, Arguments: [org.springframework.web.servlet.NoHandlerFoundException: No endpoint PUT /api/v1/escuelas/enable/.]
2024-05-14 09:11:19.752 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationErrorRutaNoMapeada(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La ruta o endpoint que desea acceder no existe, errorCode=6],[]>
2024-05-14 09:11:19.762 [main] WARN  o.s.web.servlet.PageNotFound - No mapping for PUT /api/v1/escuelas/update/
2024-05-14 09:11:19.762 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationErrorRutaNoMapeada, Arguments: [org.springframework.web.servlet.NoHandlerFoundException: No endpoint PUT /api/v1/escuelas/update/.]
2024-05-14 09:11:19.762 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationErrorRutaNoMapeada(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La ruta o endpoint que desea acceder no existe, errorCode=6],[]>
2024-05-14 09:11:19.771 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findById(..), URL: http://localhost/api/v1/escuelas/find/1, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.778 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: entityDisabledException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.EntityDisabledException: La escuela que busca se encuentra deshabilitada]
2024-05-14 09:11:19.779 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.entityDisabledException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=Ocurrio un error al realizar la operacion ya que una entidad se encuentra deshabilitada: La escuela que busca se encuentra deshabilitada, errorCode=2],[]>
2024-05-14 09:11:19.787 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.enable(..), URL: http://localhost/api/v1/escuelas/enable/9999999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.790 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 09:11:19.790 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 09:11:19.801 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.update(..), URL: http://localhost/api/v1/escuelas/update/9999999, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.804 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaNoExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaNoExistenteException: La escuela no existe en la base de datos]
2024-05-14 09:11:19.804 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaNoExistenteException(..), Respuesta: <404 NOT_FOUND Not Found,ResponseMessage[message=La escuela no fue encontrada en la base de datos, errorCode=3],[]>
2024-05-14 09:11:19.849 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: validationError, Arguments: [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaResponseDto> com.escuelas.project.escuelas_project.controllers.v1.EscuelaController.save(com.escuelas.project.escuelas_project.service.models.dtos.escuela.EscuelaDto) throws com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaExistenteException: [Field error in object 'escuelaDto' on field 'nombre': rejected value []; codes [NotBlank.escuelaDto.nombre,NotBlank.nombre,NotBlank.java.lang.String,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [escuelaDto.nombre,nombre]; arguments []; default message [nombre]]; default message [El nombre de la escuela es requerido para la creacion de una nueva escuela]] ]
2024-05-14 09:11:19.850 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.validationError(..), Respuesta: <400 BAD_REQUEST Bad Request,[ResponseMessage[message=El nombre de la escuela es requerido para la creacion de una nueva escuela, errorCode=1]],[]>
2024-05-14 09:11:19.864 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.save(..), URL: http://localhost/api/v1/escuelas/save, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.870 [main] ERROR c.e.p.e.aop.ExceptionHandlerAOP - Exception in method: escuelaExistenteException, Arguments: [com.escuelas.project.escuelas_project.service.models.exceptions.escuelaExceptions.EscuelaExistenteException: La escuela ya existe en la base de datos]
2024-05-14 09:11:19.870 [main] INFO  c.e.p.e.aop.ExceptionHandlerAOP - Método: GlobalExceptionHandler.escuelaExistenteException(..), Respuesta: <400 BAD_REQUEST Bad Request,ResponseMessage[message=No se pudo registrar el escuela con el nombre ingresado ya que la misma ya se encuentra registrada, errorCode=4],[]>
2024-05-14 09:11:19.878 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findAll(), URL: http://localhost/api/v1/escuelas/find/all, IP: 127.0.0.1, Parámetros: {}
2024-05-14 09:11:19.886 [main] INFO  c.e.p.e.aop.HttpRequestAspect - Método: EscuelaController.findAll(), Respuesta: <200 OK OK,[EscuelaResponseDto[id=1, nombre=test_d], EscuelaResponseDto[id=2, nombre=TEC 1], EscuelaResponseDto[id=3, nombre=test]],[]>
2024-05-14 09:11:19.901 [SpringApplicationShutdownHook] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
2024-05-14 09:11:19.904 [SpringApplicationShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
2024-05-14 09:11:19.915 [SpringApplicationShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
